server/config/routes.js

```diff
@@ -909,6 +909,9 @@ exports.configure = function configure(app) {
   app.get('/reports/stock/assign', stockReports.stockAssignReport);
   app.get('/reports/stock/satisfaction_rate_report', stockReports.satisfactionRateReport);
 
+  // stock api for inter-operability
+  app.get('/api/stock/rumer', stockReports.rumerApi.getData);
```

Comment:
It is good to add a route for this, but I suggest removing the /api/ part of the URL.   None of our other REST urls use it and it does not seem necessary.

Reply:
@jmcameron I am okay with your proposal to remove the part `/api` in the URL, do you have some suggestions since the route returns only data to be consumed by tiers clients?

Reply:
Will `/stock/rumer ` work?

Reply:
@jniles the function used in the API route is also used elsewhere so we do not have the freedom to make this change on the endpoint route definition without modifying other modules.